secretsstoresynccontroller/config/admissionpolicies/create_update_specific_types.yaml

```diff
@@ -0,0 +1,32 @@
+# This policy is used to deny the creation or update of secrets of types other than Opaque, 
+# kubernetes.io/basic-auth, kubernetes.io/ssh-auth and kubernetes.io/tls
+# TODO: Use parameters to allow the user to specify the name of the SecretSyncController.
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "sscontroller-create-update-secrets-types-policy"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   [""]
+      apiVersions: ["v1"]
+      operations:  ["CREATE","UPDATE"]
+      resources:   ["secrets"]
+  variables:
+  - name: typeCheck
+    expression: "(object.type == 'Opaque' || object.type == 'kubernetes.io/basic-auth' || object.type == 'kubernetes.io/ssh-auth' || object.type == 'kubernetes.io/tls')"
+  - name: userIsSecretSyncController
+    expression: "request.userInfo.username.contains('secret-store-sync-controller')"
```

Comment:
```suggestion
    expression: "request.userInfo.username == 'system:serviceaccount:NS_NAME:secret-store-sync-controller')"
```

Reply:
Yes I believe the namespace has to come from the user since it is configurable (with probably some good default).